torch/_functorch/aot_autograd.py

```diff
@@ -1114,11 +1118,11 @@ def dispatch_and_compile():
     else:
         compiled_fn = dispatch_and_compile()
 
-    if isinstance(mod, torch._dynamo.utils.GmWrapper):
-        # This function is called by the flatten_graph_inputs wrapper, which boxes
-        # the inputs so that they can be freed before the end of this scope.
-        # For overhead reasons, this is not the default wrapper, see comment:
-        # https://github.com/pytorch/pytorch/pull/122535/files#r1560096481
+    if isinstance(
+        mod, torch._dynamo.utils.GmWrapper
```

Comment:
I guess you gonna delete GmWrapper later?

Reply:
We can't unless we deprecate non-dynamo frontends who produce graphs with inputs that aren't flat